Tests/AdyenTests/Adyen Tests/DropIn/DropInActionTests.swift

```diff
@@ -41,4 +41,29 @@ class DropInActionsTests: XCTestCase {
         waitForExpectations(timeout: 15, handler: nil)
     }
 
+    func testOpenExternalApp() {
+        let config = DropInComponent.Configuration(apiContext: Dummy.context)
+        config.payment = Payment(amount: Amount(value: 100, currencyCode: "CNY"), countryCode: "CN")
+
+        let waitExpectation = expectation(description: "Expect a callback")
+        let mock = MockDropInDelegate()
```

Comment:
I would rather call it `DropInDelegateMock`, starting with `Mock` makes it seem like an imperative order or a function name :-), but its a not a big deal, we can rename it later

Reply:
fixed